src/lib/components/Swap/Status.fixture.tsx

```diff
@@ -11,7 +11,7 @@ import invariant from 'tiny-invariant'
 import { Modal } from '../Dialog'
 import { StatusDialog } from './Status'
 
-const ETH = nativeOnChain(SupportedChainId.MAINNET)
+const ETH = WRAPPED_NATIVE_CURRENCY[ChainId.MAINNET]
```

Comment:
revert. Is there a reason to prefer WRAPPED_NATIVE_CURRENCY over nativeOnChain?

Reply:
Theres no real reason here - i only did this because other places in the app use it. But nativeOnChain is probably more accurate here